encryption_works.md

```diff
@@ -220,37 +220,28 @@ With the release of OTR 4.0 in September 2012, Pidgin stopped logging OTR conver
 
 ## "Pretty Good Privacy" (PGP) Email Encryption
 
-In 1991, Phil Zimmermann developed email encryption software called [Pretty Good Privacy](https://en.wikipedia.org/wiki/Pretty_Good_Privacy), or PGP, which he intended peace activists to use while organizing in the anti-nuclear movement.
+PGP stands for "Pretty Good Privacy," software you can use to encrypt your e-mail messages. GPG is a free, open-source version of this software. The terms PGP and GPG are often used interchangeably. Unfortunately, the process is notoriously hard to use, as evidenced by Glenn Greenwald initially being 
+[unable](http://www.huffingtonpost.com/2013/06/10/edward-snowden-glenn-greenwald_n_3416978.html?1370895818) to set it up and talk security with Edward Snowden.
 
-Today, PGP is a company that sells a proprietary encryption program by the same name. [OpenPGP](http://openpgp.org/) is the open protocol that defines how PGP encryption works, and [GnuPG](http://www.gnupg.org/) (GPG for short) is free software, and is 100% compatible with the proprietary version. GPG is much more popular than PGP today because it's free for everyone to download, and cypherpunks trust it more because it's open source. The terms PGP and GPG are often used interchangably.
```

Comment:
I also like how Micah explains the difference of OpenPGP/GPG or PGP without just glossing over it. Especially since we call it PGP throughout we should include what OpenPGP is (since PGP is not actually what we use.. But maybe we can simplify it a bit.